source-browser.html

```diff
@@ -67,21 +91,21 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
         CACHE_KEY = `${ORGANIZATION}-cache`,
         EXPIRY_KEY = `${ORGANIZATION}-expiry`,
         EXPIRY_MILLIS = 60000,
-        TOPICS = ['languages', 'trunk', 'staging', 'nursery', 'incubator', 'tools'].map(topic => ({
+        LIST = ['languages', 'trunk', 'staging', 'nursery', 'incubator', 'tools'],
+        TOPICS = LIST.map(topic => ({
           topic: `apertium-${topic}`,
           name: topic,
           repoUrl: `https://github.com/${ORGANIZATION}/apertium-${topic}`,
         }));
-
       async function getReposByTopic(organization) {
         const headers = new Headers({ Accept: 'application/vnd.github.mercy-preview+json' });
         const cacheStale = !(CACHE_KEY in localStorage) || localStorage[EXPIRY_KEY] < Date.now();
-
+          
```

Comment:
There are still unnecessary whitespace additions throughout this entire file that should be deleted. Something like 10 of them.